modules/codemod/lib/v7/index.ts

```diff
@@ -1,7 +1,25 @@
-import {API, FileInfo, Options} from 'jscodeshift';
+import {Transform} from 'jscodeshift';
 
-export default function transformer(file: FileInfo, api: API, options: Options) {
+import compoundActionBar from './compoundActionBar';
+import compoundBanner from './compoundBanner';
+import recategorizeIconButtons from './recategorizeIconButtons';
+import removePropFromPopper from './removePropFromPopper';
+import renameIconPosition from './renameIconPosition';
+import renameLayoutImports from './renameLayoutImports';
+import updateSegmentedControl from './updateSegmentedControl';
+
+const transform: Transform = (file, api, options) => {
   // These will run in order. If your transform depends on others, place yours after dependent transforms
-  //   const fixes = [];
-  //   return fixes.reduce((source, fix) => fix({...file, source}, api, options), file.source);
-}
+  const fixes = [
+    compoundActionBar,
+    compoundBanner,
+    recategorizeIconButtons,
+    removePropFromPopper,
+    renameIconPosition,
+    renameLayoutImports,
+    updateSegmentedControl,
+  ];
+  return fixes.reduce((source, fix) => fix({...file, source}, api, options) as string, file.source);
+};
+
+export default transform;
```

Comment:
I have no idea why none of these transforms were hooked up yet

Reply:
https://www.youtube.com/watch?v=0GypdsJulKE

Reply:
I have a pretty good idea why. I'll think about how we can prevent it from slipping again.

Reply:
One way is to update all codemods to import from this index file instead of their individual transform file. Then the spec won't work until everything is hooked up correctly.

I think we did it this way in v5 because we had cascading transforms that we wanted to isolate, but that might be a mistake. It makes it too easy to have all tests pass, but none of the codemods actually work.